webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/nodes/NodeInfoService.scala

```diff
@@ -507,10 +507,9 @@ trait NodeInfoServiceCached extends NodeInfoService with NamedZioLogger with Cac
   override final def getDeletedNodeInfo(nodeId: NodeId): Box[Option[NodeInfo]] = getNotAcceptedNodeInfo(nodeId, RemovedInventory).toBox
 
   /**
-   * Clear cache. Try a reload asynchronously, disregarding
-   * the result
+   * Clear cache.
    */
-  override def clearCache(): Unit = this.synchronized {
```

Comment:
why no semaphore here ??

Reply:
It was the only `synchronized` in the whole class, and it does not have anything to synchronize. I believe it's a scare from past battles.